platforms/blabsy/src/components/input/input.tsx

```
@@ -26,262 +26,265 @@
+    const [loading, setLoading] = useState(false);
+    const [visited, setVisited] = useState(false);
+
+    const { user, isAdmin } = useAuth();
+    const { name, username, photoURL } = user as User;
+
+    const inputRef = useRef<HTMLTextAreaElement>(null);
+
+    const previewCount = imagesPreview.length;
+    const isUploadingImages = !!previewCount;
+
+    useEffect(
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: modal

This dependency is not specified in the hook dependency list.

Either include it or remove the dependency array

(lint/correctness/useExhaustiveDependencies)